app/src/main/java/com/mycompany/myapp/ui/BaseGlideModule.kt

```diff
@@ -0,0 +1,42 @@
+package com.mycompany.myapp.ui
+
+import android.databinding.BindingAdapter
+import android.graphics.drawable.Drawable
+import android.widget.ImageView
+import com.bumptech.glide.annotation.GlideModule
+import com.bumptech.glide.module.AppGlideModule
+import com.bumptech.glide.request.RequestOptions
+
+object GlideBindingAdapter {
+    @JvmStatic
+    @BindingAdapter(
+            value = [
+                "url",
+                "placeholder",
+                "fallback",
+                "error"],
+            requireAll = false)
+    fun setImageSource(
+            view: ImageView,
+            url: String?,
+            placeholder: Drawable?,
+            fallback: Drawable?,
+            error: Drawable?) {
+        GlideApp.with(view.context)
+                .load(url)
+                .apply(RequestOptions()
+                        .fitCenter()
```

Comment:
This works well for things that need this option, is there any way we could this configurable?

Reply:
Good thought. I didn't look too far into it, but I have an idea